README.md

```diff
@@ -1 +1,412 @@
-# Writing Initializers
\ No newline at end of file
+# Writing Initializers
+
+## Objectives
+
+1. Recognize when initializers are useful, especially with read-only properties.
+2. Identify the role of a designated initializer.
+3. Compose a designated initializer.
```

Comment:
"Compose" should probably be avoided unless you mean function composition. "Write."